save-api/src/main/kotlin/com/saveourtool/save/api/SaveCloudClientEx.kt

```diff
@@ -2,7 +2,7 @@
 
 package com.saveourtool.save.api
 
-import com.saveourtool.save.agent.TestExecutionDto
+import com.saveourtool.save.agent.TestExecutionExDto
```

Comment:
Also it might be better to call it `TestExecutionExtDto` as far as `ext` is usually short for `extended` imo

Reply:
Renamed to `TestExecutionExtDto` in cf3d3a453d455beeb84b178cf78b4e6d55fd8184.